verde/utils.py

```diff
@@ -201,3 +202,93 @@ def maxabs(*args):
     arrays = [np.atleast_1d(i) for i in args]
     absolute = [np.abs([i.min(), i.max()]).max() for i in arrays]
     return np.max(absolute)
+
+
+def grid_to_table(grid):
+    """
+    Convert a grid to a table with the values and coordinates of each point.
+
+    Takes a 2D grid as input, extracts the coordinates and runs them through
+    :func:`numpy.meshgrid` to create a 2D table. Works for 2D grids and
+    n-dimensional variables. Use cases includes passing gridded data to
```

Comment:
```suggestion
    any number of variables. Use cases includes passing gridded data to
```

The "variables" are 2D but there can be any number of them.